package.json

```diff
@@ -30,6 +30,7 @@
     "configurationModule": "lib/imperative.js"
   },
   "dependencies": {
+    "@zowe/cli": "^3.2.0",
```

Comment:
I'm not sure this should be here. I think we just wanted it listed a dev dependency and peer dependency as this is what the imperative plugin installer expects.

Reply:
Okay, I'll try that. We do need the newer version of Zowe at runtime of course, it's not just a build dependency.

Reply:
Okay, moving @zowe/cli to be a dev dependency works, but @zowe/imperative and @brightside/core are needed as full dependencies or the build fails.

Reply:
That doesn't seem right to me. Have a look at https://github.com/zowe/zowe-cli-cics-plugin/pull/26/files where they made a similar change to the CICS plugin.

Reply:
The zowe plugins have the advantage of being able to reference files across projects using relative references, we have to do so via the global namespace. I don't know if that explains the difference, but I couldn't just copy zowe code into our plugin as the relative imports often wouldn't resolve.

Reply:
I have a set of changes which sort this out, I'll submit a separate PR.